lib/image-diff.js

```diff
@@ -79,7 +79,7 @@ ImageDiff.createDiff = function (options, cb) {
     // DEV: According to http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=17284
     // DEV: These values are the total square root mean square (RMSE) pixel difference across all pixels and its percentage
     // TODO: This is not very multi-lengual =(
-    var resultInfo = stderr.match(/all: (\d+\.?\d*) \((\d+\.?\d*)\)/);
+    var resultInfo = stderr.match(/all: (\d+(?:\.\d+)?) \((\d+(?:\.\d+)?(?:[Ee]-?\d+)?)\)/);
```

Comment:
This regexp grew a lot in complexity. Why are we moving to `?:` for some of the groups?